Go/Go.sublime-syntax

```diff
@@ -719,11 +719,20 @@ contexts:
 
   match-keyword-break:
     - match: \bbreak\b
-      scope: keyword.control.go
+      scope: keyword.control.flow.break.go
 
   match-keyword-case:
     - match: \bcase\b
-      scope: keyword.control.go
+      scope: keyword.control.switch.case.go
+      push: match-case-value
+
+  match-case-value:
+    - match: '{{predeclared_type}}'
+      scope: support.type.go
+    - match: ':'
+      scope: punctuation.separator.case-statement.go
+      pop: true
+    - include: match-any
```

Comment:
```suggestion
    - match: ':'
      scope: punctuation.separator.case-statement.go
      pop: true
    - match: '{{predeclared_type}}'
      scope: support.type.go
    - include: match-any
```

Maybe just a personal habit, but I tend to organize popping patterns at the top of a context.